commands/apps_dev.go

```diff
@@ -375,10 +402,21 @@ func RunAppsDevBuild(c *CmdConfig) error {
 	} else if res.ExitCode == 0 {
 		template.Buffered(
 			textbox.New().Success(),
-			`{{success checkmark}} successfully built {{success .img}} in {{highlight (duration .dur)}}`,
+			heredoc.Doc(`
+				{{success checkmark}} successfully built {{success .component}} in {{highlight (duration .dur)}}
+				{{success checkmark}} created container image {{success .img}}
+				
+				{{pointerRight}} push your image to a container registry using {{highlight "docker push"}}
+				{{pointerRight}} or run it locally using {{highlight "docker run"}}; for example:
+				  
+				   {{muted promptPrefix}} {{highlight (printf "docker run --rm -p 8080:8080 %s" .img)}}
+				
+				  then access your component at {{underline "http://localhost:8080"}}`,
```

Comment:
:ok_hand: This is a good addition. Was going to mention that the post-build experience left a lot of the user to figure out on their own.